src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,48 +3,150 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size;
+    private Node<T> head;
+    private Node<T> tail;
+
     @Override
     public void add(T value) {
-
+        if (head != null) {
+            Node<T> newTail = new Node(value, tail, null);
+            tail.next = newTail;
+            tail = newTail;
+        } else {
+            head = new Node(value, null, null);
+            tail = head;
+        }
+        size++;
     }
 
     @Override
     public void add(T value, int index) {
-
+        if (index > size || index < 0) {
+            throw new IndexOutOfBoundsException();
+        }
+        if (index == size) {
+            Node<T> previous = tail;
```

Comment:
we can call method add(T t) here

Reply:
Fixed.  The method add(value) is there.  Thank you.